src/neo/SmartContract/ContractParametersContext.cs

```diff
@@ -180,14 +179,15 @@ public bool AddSignature(Contract contract, ECPoint pubkey, byte[] signature)
             }
         }
 
-        private ContextItem CreateItem(Contract contract)
+        private ContextItem CreateItem(Contract contract, bool copyScript = true)
```

Comment:
Even through I don't like this `copyScript` parameter, but it works well.

Reply:
Otherwise we will copy the full script inside the witness.

Reply:
You can create a subclass of `Contract`, and set it's `Script` to `null`.

Reply:
Done